.pre-commit-config.yaml

```diff
@@ -145,6 +145,24 @@ repos:
         pass_filenames: false
         require_serial: true
         files: ^.github/workflows/.*$
+      - id: check-log-usage
+        name: Check for proper log usage (use llama_stack.log instead)
+        entry: bash
+        language: system
+        types: [python]
+        pass_filenames: true
+        args:
+          - -c
+          - |
+            matches=$(grep -EnH '^[^#]*\b(import logging|from logging\b)' "$@" | grep -v '# allow-direct-logging' || true)
```

Comment:
you could make the regexp more robust by allowing an arbitrary amount of spaces like with `\s+`. Also you might want allow `#allow-direct-logging` and `#  allow-direct-logging`, too (so maybe grep -v -e '#\s*allow-direct-logging')